verilog/parser/verilog_parser_unittest.cc

```diff
@@ -5774,6 +5806,10 @@ TEST(VerilogParserTestMatchAll, RandSequence) {
   TestVerilogParserMatchAll(kRandSequenceTests);
 }
 
+TEST(VerilogParserTestMatchAll, NetAlias) {
```

Comment:
Somewhere around line 5620, don't forget to add:
`TestVerilogParser(kNetAliasTests);`

Reply:
Absolutely right, added.

Thanks for the review!